templates/definition/meter/sonnenbatterie.yaml

```
@@ -26,6 +26,18 @@ params:
       de: API Token (benötigt für aktive Batteriesteuerung)
       en: API Token (required for active battery control)
     usages: ["battery"]
+  - name: backupbuffer
+    required: false
+    help:
+      de: In der Batterie verbleibende Reserve-Kapazität (in %)
+      en: default remaining battery charge (in %)
+    default: 10
+  - name: chargingbuffer
+    required: false
+    help:
+      de: Maximaler Entladestand der Batterie während des evcc-Ladevorgangs
+      en: Maximum discharge level of the SonnenBatterie during vehicle charging
+    default: 50
```

```suggestion
    help:
      de: Verbleibende Kapazität in der Batterie beim Laden des Fahrzeugs (in %)
      en: Remaining capacity in the battery during vehicle charging (in %)
    default: 100
```
* I suggest using `100` as default in order to have the same behaviour as before (default: do not discharge battery) without configuring additional parameters.
* Adding `usages: ["battery"]` is showing this parameter only in the battery documentation.
* Minor rewording of help text

---

Using 100 as a default may be dangerous since another user wrote that setting the value higher than the current load of the battery will make the battery start charging. I just tested with my setup and my battery **didn't** start charging. Would be nice if you could verify with yours. Other than that: no objection there.

---

Ok, I did some more testing and for my setup it is like this:
- if the SonnenBatterie is in mode 1 (Manual) or mode 10 (ToU) setting EM_USOC to a value above the current charge level does nothing
- if, however, the SonnenBatterie is in mode 2 (Auto) setting EM_USOC to a value above the current charge level will make the battery start charging, no matter whether there's enough PV energy available or not.

---

> Ok, I did some more testing and for my setup it is like this:
> 
>     * if the SonnenBatterie is in mode 1 (Manual) or mode 10 (ToU) setting EM_USOC to a value above the current charge level does nothing
> 
>     * if, however, the SonnenBatterie is in mode 2 (Auto) setting EM_USOC to a value above the current charge level will make the battery start charging, no matter whether there's enough PV energy available or not.

Thank you for testing this further. I can reproduce it with my sonnenBatterie eco 8.

From my point of view, the second point disqualifies the use of EM_USOC together with mode 2 (auto/self consumption). We have a good chance of charging not only the car from the grid, but also our home battery!